openedx/adg/lms/applications/models.py

```diff
@@ -193,6 +193,20 @@ def is_user_business_line_admin(cls, user):
         """
         return cls.objects.filter(group__user=user).exists()
 
+    @classmethod
+    def get_business_lines_for_user(cls, given_user):
```

Comment:
no need for specifying `get` in function name as `business_lines_for_user` is enough and also `given_user` can be replaced with `user` simply.